skyportal/handlers/api/internal/robotic_instruments.py

```diff
@@ -0,0 +1,13 @@
+from ....models import Instrument
+from baselayer.app.access import auth_or_token
+from ...base import BaseHandler
+
+
+class RoboticInstrumentsHandler(BaseHandler):
+    @auth_or_token
+    def get(self):
+        instruments = Instrument.query.filter(
+            Instrument.api_classname != None  # noqa
```

Comment:
why the != None?

Reply:
SQLalchemy requires an explicit declaration of != None to render the appropriate filter in SQL.

Reply:
This is a Python expression, so what control does SQLAlchemy have over it?

Reply:
There's also, `query.filter(Cls.col.isnot(None)))`

Reply:
changed to this.